src/main/java/org/apache/arrow/flight/spark/FlightClientOptions.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2019 Ryan Murray
```

Comment:
want to update the license header to include you or just `authors`

Reply:
@rymurr Kyle included you, himself, and Doron https://github.com/rymurr/flight-spark-source/pull/8/commits/55f804a933c1beeb7b8731756f047b73f5dd95b0